query-service-impl/src/main/java/org/hypertrace/core/query/service/postgres/TableDefinition.java

```diff
@@ -124,6 +132,8 @@ public static TableDefinition parse(
         spec = new PostgresColumnSpec(physName, ValueType.BYTES, false);
       } else if (tdigestFields.contains(physName)) {
         spec = new PostgresColumnSpec(physName, ValueType.STRING, true);
+      } else if (arrayFields.contains(physName)) {
```

Comment:
Note: Currently, we have only `array<string>` in avdl files, so this will suffice for now. But, we will have to extend the same logic for any other array types (`array<double>`, array<int>`), etc.

Reply:
We will add support as and when needed

Reply:
Lets have backlog ticket for this.